src/models/event.js

```diff
@@ -299,6 +310,100 @@ utils.extend(module.exports.MatrixEvent.prototype, {
     },
 
     /**
+     * Check if this event is currently being decrypted.
+     *
+     * @return {boolean} True if this event is currently being decrypted, else false.
+     */
+    isBeingDecrypted: function() {
+        return this._decrypting;
+    },
+
+    /**
+     * Start the process of trying to decrypt this event.
+     *
+     * (This is used within the SDK: it isn't intended for use by applications)
+     *
+     * @internal
+     *
+     * @param {module:crypto} crypto crypto module
+     */
+    attemptDecryption: function(crypto) {
+        if (!crypto) {
+            this._badEncryptedMessage("Encryption not enabled");
+            return;
+        }
+
+        if (!this.isEncrypted()) {
+            throw new Error("Attempt to decrypt event which isn't encrypted");
```

Comment:
the former sounds like somebody using the API incorrectly. The second happens whenever you receive an encrypted event but haven't loaded `olm.js` (because you're in Syria, or whatever)